packages/api/src/sequelize/migrations-shared.ts

```diff
@@ -87,3 +87,31 @@ export const createTable = async (
   );
   await addUpdatedAtTrigger(queryInterface, options.transaction, tableName);
 };
+
+export const resultsColumns = {
```

Comment:
I suggest we don't add anything to shared files when it comes to migrations, they should be as atomic/encapsulated as possible - this was a bad pattern we started in the beginning.

To see the problem, imagine we change this in a few month's time, removing `status`. Then the migration we're creating here on this PR will run with that new code when someone tries to populate their file in the future, which might result in a migration error.